.github/workflows/integration-test.yaml

```diff
@@ -14,22 +14,36 @@ jobs:
   check:
     runs-on: macos-latest
 
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+
+          - platform: iOS
```

Comment:
Can we keep the version numbers in the `platform` property so that they appear alongside the CI job, like they did before?

Reply:
I guess this value is used as a var later - ${{ matrix.platform }}

Reply:
maybe rename a lane to just "iOS_XXX"? "test_" prefix occupies too much space in GH UI.

Reply:
I think it's best to keep the `test_` prefix. Perhaps we call the param `lane_suffix` and then do `lane: test_${{ matrix.lane_suffix }}`?

Reply:
Oh, sorry, I was confused, I didn't notice the `lane` param. Ignore this thread.